language/stdlib/modules/association.move

```diff
@@ -0,0 +1,112 @@
+// Implements logic for registering addresses as association addresses, and
+// determining if the sending account is an association account.
+// Errors:
+// 1000 -> INVALID_GENESIS_ADDR
+// 1001 -> INSUFFICIENT_PRIVILEGES
+// 1002 -> NOT_AN_ASSOCIATION_ACCOUNT
+// 1003 -> ACCOUNT_DOES_NOT_HAVE_PRIVILEGE
+// 1004 -> ACCOUNT_DOES_NOT_HAVE_PRIVILEGE_RESOURCE
+address 0x0:
+
+module Association {
+    use 0x0::Transaction;
+
+    // The root account privilege. This is created at genesis and has
+    // special privileges (e.g. removing an account as an association
+    // account). It cannot be removed.
+    resource struct Root { }
+
+    // There are certain association capabilities that are more
+    // privileged than other association operations. This resource with the
+    // type representing that privilege is published under the privileged
+    // account.
+    resource struct PrivilegedCapability<Privilege> { is_certified: bool }
+
+    // A type tag to mark that this account is an association account.
+    // It cannot be used for more specific/privileged operations.
```

Comment:
Did we end up deciding what privileges this confers (other than the ability to acquire `PrivilegedCapability`'s)?

My understanding of where we left this is that: 
- it's useful for revoking all `PrivilegedCapability`'s at once, but we could/perhaps would want to use `freeze` for this
- we should keep this around if we determine there some other privs that all association accounts have, but not otherwise.

Reply:
Yep! Exactly right. The main reason (the way I see it) is for the second case--specifically for prioritization of transactions.

Reply:
Ahhh got it I forgot about this! Might be useful to note in the comment that this is the only privilege this confers.